pkg/api/registry.go

```diff
@@ -36,4 +40,5 @@ type DevfileStackVersion struct {
 	IsDefault       bool     `json:"isDefault"`
 	SchemaVersion   string   `json:"schemaVersion,omitempty"`
 	StarterProjects []string `json:"starterProjects"`
+	CommandGroups   map[schema.CommandGroupKind]bool
```

Comment:
To follow the field naming convention in the JSON output:

```suggestion
	CommandGroups   map[schema.CommandGroupKind]bool `json:"commandGroups,omitempty"`
```